01_oop_and_ood/src/Triangle.ts

```diff
@@ -0,0 +1,52 @@
+import { Point } from './Point';
+import { Shape } from './Shape';
+
+export class Triangle extends Shape {
+    constructor(point1: Point, point2: Point, point3: Point)
+    constructor(point1: Point, point2: Point, point3: Point, color: string, filled: boolean)
+    constructor(...args: any[]) {
```

Comment:
Same here. Please, review previous comments to previous constructors and fix this one.